app/src/main/java/org/wikipedia/analytics/NotificationPreferencesFunnel.kt

```diff
@@ -32,8 +36,26 @@ class NotificationPreferencesFunnel(app: WikipediaApp) : Funnel(app, SCHEMA_NAME
         }
     }
 
+    fun logNotificationFilterPrefs() {
+        val fullFiltersList = mutableListOf<String>()
+        val toggleMap = HashMap<String, Boolean>()
+        val filteredList = StringUtil.csvToList(Prefs.notificationsFilterLanguageCodes.orEmpty())
+        fullFiltersList.addAll(allWikisList())
+        fullFiltersList.addAll(allTypesIdList())
```

Comment:
Would it be possible to have these to:

`NotificationsFilterActivity.allWikisList()` and  `NotificationsFilterActivity.allTypesIdList()`?